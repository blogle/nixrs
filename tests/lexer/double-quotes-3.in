"\a\b\c\\\"\n\r\t\${}"
