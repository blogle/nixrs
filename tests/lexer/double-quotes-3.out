[1:1-1:2] (Quote(Normal)) "\""
[1:2-1:22] (StrPart(Normal)) "\\a\\b\\c\\\\\\\"\\n\\r\\t\\${}"
[1:22-1:23] (Quote(Normal)) "\""
